### PR TITLE
Distribute licenses and devel node_modules

### PR DESCRIPTION
**Pull request by stefwalter**
This is a initial commit to distribute devel reqs and licenses. However it makes a file that's too big, I'd like to split out the node_modules/ distribution before we include this in Cockpit.

CC @mbiebl @mvollmer @fsateler

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -231,10 +239,11 @@ upload-pot:
 download-po:
 	zanata-cli -B pull --min-doc-percent 30 --project-config "$(srcdir)/tools/zanata.xml" -s "$(builddir)/po" -t "$(srcdir)/po"
 
-BOWER = $(srcdir)/tools/bower_components
+BOWER = $(srcdir)/bower_components
 
 update-bower:
-	( cd $(srcdir)/tools && ./bower update )
+	bower update
```
Comment (dperpeet) [inline Makefile.am]: we need to change to the source directory here
https://github.com/dperpeet/cockpit/commit/b9c05963681258442a3892b912f4afd680afc912

**Comment by stefwalter** [comment]
To do this we need to make two other changes:

     * Use xz to compress the tarball, so that it stays relatively small
     * Use the tar-pax format because file names get very long.

**Comment by dperpeet** [comment]
I need to look into this further, but update-lib fails for me:
```
../cockpit/tools/missing ../cockpit/tools/jsmodule -m -d jquery -o ../cockpit/pkg/base1/bootstrap-datepicker.js \
    ../cockpit/bower_components/patternfly/components/bootstrap-datepicker/dist/js/bootstrap-datepicker.js
../cockpit/tools/missing ../cockpit/tools/jsmodule -m -d jquery -o ../cockpit/pkg/base1/bootstrap-datepicker.min.js \
    ../cockpit/bower_components/patternfly/components/bootstrap-datepicker/dist/js/bootstrap-datepicker.min.js
../cockpit/tools/missing ../cockpit/tools/jsmodule -m -d jquery -o ../cockpit/pkg/base1/bootstrap-combobox.js \
    ../cockpit/bower_components/patternfly/components/bootstrap-combobox/js/bootstrap-combobox.js
../cockpit/bower_components/patternfly/components/bootstrap-combobox/js/bootstrap-combobox.js: ENOENT, open '../cockpit/bower_components/patternfly/components/bootstrap-combobox/js/bootstrap-combobox.js'
Makefile:8992: recipe for target 'update-lib' failed
```

**Comment by stefwalter** [comment]
@dperpeet Fixed that issue. But you may need to ```rm -rf bower_components/patternfly``` before ```make update-lib``` to get bower to recognize it as fixed.

**Pull request closed without merging**